src/modules/AccountSettings/SettingOg/SimpleMode.tsx

```diff
@@ -180,7 +180,7 @@ export default function SimpleMode({
               src={`${BASEURL}/api/og?type=custom-user&slug=${owner?.slug}&theme=${activeGradient}&text=${
                 watchTextOgPublik ||
                 'Kumpulkan pertanyaan anonim dengan lebih mudah'
-              }&r=${new Date().getTime()}`}
+              }&r=${new Date().getTime()}&forceSimpleMode=true`}
```

Comment:
Required so the simple preview will still work when a previous advanced configuration is found.